GVRf/Framework/src/org/gearvrf/GVRRenderData.java

```diff
@@ -325,7 +362,34 @@ public void setRenderingOrder(int renderingOrder) {
      *         not.
```

Comment:
Ok. Did changes but still have one concern. I did not had the care to keep and deprecate changes on native side. So, some interfaces changed. (such as culltest methods)

Do you want me to review interface changes on native?